api/database/data/qa/01_non_log_data.sql

```diff
@@ -0,0 +1,160 @@
+----INSERT ORGS----
+INSERT INTO public.organisation(
```

Comment:
🔥 public

Reply:
Doesn't really matter, all tables are in the public schema